chef/cookbooks/monasca/recipes/role_monasca_server.rb

```diff
@@ -16,5 +16,6 @@
 
 if CrowbarRoleRecipe.node_state_valid_for_role?(node, "monasca", "monasca-server")
   include_recipe "monasca::server"
-#  include_recipe "monasca::api"
+  include_recipe "monasca::api"
```

Comment:
If we plan on keeping it that way, I think that should go before monasca::server if the purpose of those recipes is just to register stuff in keystone.